codex/slots/proofs/prover.nim

```diff
@@ -86,16 +87,15 @@ proc verify*(
   inputs: AnyProofInputs): Future[?!bool] {.async.} =
   ## Prove a statement using backend.
   ## Returns a future that resolves to a proof.
-
   self.backend.verify(proof, inputs)
 
 proc new*(
   _: type Prover,
   store: BlockStore,
   backend: AnyBackend,
-  nSamples: int): Prover =
+  config: CodexConf): Prover =
```

Comment:
Just pass the number of samples instead of the config object, there is no need for it here.

Reply:
in other words, revert back to original.